node/mock-test.ts

```diff
@@ -5,19 +5,14 @@ import os = require('os');
 import path = require('path');
 import cmdm = require('./taskcommand');
 import shelljs = require('shelljs');
-import syncRequest from 'sync-request';
+const Downloader = require("nodejs-file-downloader");
 
 const COMMAND_TAG = '[command]';
 const COMMAND_LENGTH = COMMAND_TAG.length;
 const downloadDirectory = path.join(process.env.HOME || process.env.HOMEPATH || process.env.USERPROFILE as string, 'azure-pipelines-task-lib', '_download');
 
 export class MockTestRunner {
-    constructor(testPath: string, taskJsonPath?: string) {
-        this._taskJsonPath = taskJsonPath || '';
-        this._testPath = testPath;
-        this.nodePath = this.getNodePath();
-    }
-
+    constructor() { }
```

Comment:
This looks breaking. We should keep backward compatibility with previous api

Reply:
restored old style constructor